sdk/client/client.go

```diff
@@ -672,6 +672,18 @@ func (c *Client) retryableClient(checkRetry retryablehttp.CheckRetry) (r *retrya
 	r.CheckRetry = checkRetry
 	r.ErrorHandler = RetryableErrorHandler
 	r.Logger = log.Default()
+	r.RetryWaitMin = 1 * time.Second
+	r.RetryWaitMax = 61 * time.Second
```

Comment:
FWIW some [ARM API's do have `Retry-After` headers with an excessive timeout (e.g. 10m)](https://github.com/Azure/azure-rest-api-specs/issues/13115) - but I don't see any harm in us polling more regularly given that extended interval so 👍

Reply:
Oh if the response includes a `Retry-After` header, that is always selected first before falling back to exponential backoff (i.e. these are not used in that case)